parteh/PRTParametersMod.F90

```diff
@@ -100,7 +100,8 @@ module PRTParametersMod
 
      real(r8), allocatable :: c2b(:)                        ! Carbon to biomass multiplier [kg/kgC]
      real(r8), allocatable :: wood_density(:)               ! wood density  g cm^-3  ...
-     real(r8), allocatable :: woody(:)                      ! Does the plant have wood?      (1=yes, 0=no)
+
+     integer , allocatable :: woody(:)                      ! Does the plant have wood?      (1=yes, 0=no)
```

Comment:
I like switching this to an integer, but I'm surprised this worked, without modifying the read command.  This is how its coded for the "evergreen" integer switch:

https://github.com/NGEET/fates/blob/f8e16e2c53eb3226b33417fc6b94d7873cf2cf00/parteh/PRTParamsFATESMod.F90#L409-L414